pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -108,25 +111,42 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 	}
 
 	if tr.IsDone() {
+		var merr *multierror.Error
```

Comment:
hmm interesting!

Reply:
might be worth a comment explaining why we are using multierror in this case, when we don't elsewhere? (reading through im thinking its b/c handling cloudevents is kind of "in addition" to our "main" logic, so we want to track errors related to that but still continue execution?)

Reply:
Added a comment